changelog.txt

```diff
@@ -3,6 +3,7 @@
 = 2.3.0 - 2021-xx-xx =
 * Add - Introduced deposit currency filter for transactions overview page.
 * Add - Download transactions report in CSV.
+* Update - Change the default track `recordEvent` to use @woocommerce/tracks.
```

Comment:
Could you please add the same change to `readme.txt`?

Reply:
Done in 73101ab